python/notebooks/spanner/README.md

```diff
@@ -0,0 +1,46 @@
+## Jupyter Notebook Solution for migrating MySQL database to lLoud Spanner using Dataproc Templates
+
+Template for reading tables from HIVE Database and writing to BigQuery Dataset.
+
+Refer [Setup Vertex AI - PySpark](./../README.md) to setup new Jupyter notebook in vertexAI. Once the setup is done navigate to 
+[dataproc-templates/python/notebooks/spanner](https://github.com/GoogleCloudPlatform/dataproc-templates/blob/mysql-to-spanner-nb/python/notebooks/spanner/) folder and open [mysql_to_spanner.ipynb](https://github.com/GoogleCloudPlatform/dataproc-templates/blob/mysql-to-spanner-nb/python/notebooks/spanner/mysql-to-spanner.ipynb) notebook.
```

Comment:
Change file name to the current naming convention: MySqlToSpanner_vertex_pipeline_pyspark.ipynb